.vib/oauth2-proxy/runtime-parameters.yaml

```diff
@@ -11,7 +11,7 @@ configuration:
     oidc_issuer_url = "http://vmware-oauth2-proxy.my:5556/dex"
     redirect_url = "http://vmware-oauth2-proxy.my/oauth2/callback"
     cookie_secure = false
-    skip_auth_routes = [ "^\/dex\/.*" ]
+    skip_auth_routes = [ "^/dex/.*" ]
```

Comment:
It's not necessary to scape the slash character according to the documentation

https://github.com/oauth2-proxy/oauth2-proxy/blob/master/contrib/oauth2-proxy.cfg.example#L66